index.php

```diff
@@ -25,9 +25,8 @@
 <?php
 require_once __DIR__ . '/translations.php';
 ?>
-
 <!DOCTYPE html>
-<html>
+<html lang="<?php if(isset($_GET['lang'])) { echo $_GET['lang']; } else { echo 'en'; } ?>" dir="<?php if(isset($_GET['lang'])) { echo 'rtl'; } else { echo 'ltr'; } ?>">
```

Comment:
Looking at this, if lang is not set it will use right to left, english is the default language?

Reply:
You probably want to do it so only certain languages use the rtl feature.

Reply:
Oh, I didn't pay attention and I left the code incomplete, I'll correct it soon. I just wanted to change the direction when it was Persian. I'm so sorry.